vmdb/Gemfile

```diff
@@ -9,16 +9,15 @@ MiqBundler.include_gemfile("../lib/Gemfile", binding)
 
 gem "jquery-rjs", "=0.1.1", :git => 'https://github.com/amatsuda/jquery-rjs.git'
 gem 'angularjs-rails', '=1.2.4'
-gem 'jquery-rails', "~>2.2.2"
+gem 'jquery-rails', "=2.2.2"
 gem 'jquery-hotkeys-rails'
 gem 'codemirror-rails', "=4.2"
 
 # On MS Windows run "bundle config --local build.libv8 --with-system-v8" first
 
 gem "sprockets-sass",  "~>1.2.0"
-gem "sprockets-less",  "~>0.6.1"
-gem 'therubyracer', :require => false, :platform => :ruby
```

Comment:
@Jprause Removing therubyracer will simplify the build process. So this is a good thing, but will require changes to the automated build script, which I'd be glad to help with.
So the build team is very happy to see therubyracer removed! :+1: